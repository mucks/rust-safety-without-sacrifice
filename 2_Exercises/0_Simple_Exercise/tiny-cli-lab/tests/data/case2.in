# Mixed spacing and larger numbers
ADD 100 250
MUL 12 12
REV ab cd ef
UPPER rust
COUNT Mississippi
