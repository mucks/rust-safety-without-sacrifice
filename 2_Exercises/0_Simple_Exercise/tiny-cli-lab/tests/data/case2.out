350
144
ef cd ab
RUST
4
